Formula/podman.rb

```diff
@@ -0,0 +1,27 @@
+class Podman < Formula
+  desc "Tool for managing OCI containers and pods"
+  homepage "https://podman.io/"
+  url "https://github.com/containers/podman/archive/v2.0.5.tar.gz"
+  sha256 "cd7618429f02db8e7dbca29c6f8a7623a91a01804ef768dc5ec8c7660679b0c5"
+  license "Apache-2.0"
+
+  depends_on "go-md2man" => :build
+  depends_on "go@1.14" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    system "make", "podman-remote-darwin"
```

Comment:
```suggestion
    system "make", "podman-remote"
```

Could this be more os-agnostic? Homebrew supports Linux too and we are actively looking to reduce the diff between homebrew-core and linuxbrew-core. That change would certainly help.

Reply:
I think there are build difference between the podman-remote-darwin make target and the podman-remote make target. podman-remote has dependencies on the linux kernel and linux architecture, and podman-remote-darwin makes sure those features are taken out so that it builds and runs correctly on a mac.

Reply:
Pity, but okay.